tests/framework-tests/preferences/html/Panels-test.html

```diff
@@ -53,62 +53,27 @@ <h2 id="qunit-userAgent"></h2>
-            <input type="radio" class="flc-prefsEditor-themeInput fl-hidden-accessible" name="theme" id="default" value="default" />
-            <div class="fl-indicator"></div>
-            <label for="default" class="flc-prefsEditor-theme-label">
-                <span class="fl-preview-A"></span>
-                <span class="fl-hidden-accessible"></span>
-                <span class="fl-crossout"></span>
-            </label>
-        </div>
+        <!-- HTML markup for corresponding panel will be injected here -->
     </div>
 
     <div class="flc-textSize">
-        <ul class="fl-prefsEditor-topContainer fl-clearfix">
-            <li class="flc-prefsEditor-min-text-size">
-                <label for="min-text-size">Text Size</label>
-                <div class="fl-inputs">
-                    <div class="fl-force-left">
-                        <div class="fl-force-left fl-text-slider-left"><span class="fl-hidden-accessible">icon of a small capital letter 'A'</span></div>
-                        <div class="flc-textfieldSlider-slider fl-force-left fl-slider fl-slider-horz"> </div>
-                        <div class="fl-force-left fl-text-slider-right"><span class="fl-hidden-accessible">icon of a large capital letter 'A'</span></div>
-                    </div>
-                    <div class="fl-slider-input"><input id="min-text-size" class="flc-textfieldSlider-field" type="text" /> times</div>
-
-                </div>
-            </li>
-        </ul>
+        <!-- HTML markup for corresponding panel will be injected here -->
     </div>
 
     <div class="flc-lineSpace">
-        <ul class="fl-prefsEditor-topContainer fl-clearfix">
-            <li class="flc-prefsEditor-line-space">
-                <label for="line-space">Line Space</label>
-                <div class="fl-inputs">
-                    <div class="fl-force-left">
-                        <div class="fl-force-left fl-spacing-slider-left"><span class="fl-hidden-accessible">icon of 3 horizontal lines with narrow spacing</span></div>
-                        <div class="flc-textfieldSlider-slider fl-force-left fl-slider fl-slider-horz"> </div>
-                        <div class="fl-force-left fl-spacing-slider-right"><span class="fl-hidden-accessible">icon of 3 horizontal lines with wide spacing</span></div>
-                    </div>
-                    <div class="fl-slider-input"><input id="line-space" class="flc-textfieldSlider-field" type="text" /> times</div>
-
-                </div>
-            </li>
-        </ul>
+        <!-- HTML markup for corresponding panel will be injected here -->
     </div>
 
     <div class="flc-layout">
-        <input type="checkbox" id="toc-choice" class="flc-prefsEditor-toc fl-force-left" />
+        <!-- HTML markup for corresponding panel will be injected here -->
     </div>
 
     <div class="flc-links">
+         <!-- Unable to automate this composite pane successfully. See FLUID-5726  -->
```

Comment:
Maybe say something like "Unable to inject markup for this composite pane successfully. See FLUID-5726" that way it is more inline with the terminology you used for the other comments above.